src/PolyhedralGeometry/PolyhedralComplex/standard_constructions.jl

```diff
@@ -0,0 +1,91 @@
+# Workaround for turning a PolyhedralFan of polymake into a proper PolyhedralComplex
+function polyhedral_complex_workaround(pm::Polymake.BigObject)
+    pc = pm
+    typename = Polymake.type_name(pm)
+    if typename[1:13] == "PolyhedralFan"
```

Comment:
This should be refactored throughout: test for an exact match (not a substring).

Reply:
It is a workaround that goes away with the next polymake release, once 4.6 is in Julia.